### ISSUE TITLE
Missing stream properties for entities where the entities are in the base type.

### ISSUE THREAD

**Issue by darrelmiller** [microsoftgraph/msgraph-sdk-powershell#736]
![image](https://user-images.githubusercontent.com/447694/124518731-34d31580-ddb5-11eb-9df9-3fe926ea658c.png)
We generate Get-OrganizationBranding but not any commands for the stream properties:
![image](https://user-images.githubusercontent.com/447694/124518774-59c78880-ddb5-11eb-9a7f-e175143a14fc.png)
 
[AB#10131](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/10131)

**Comment by peombwa** [comment]
Fixed by https://github.com/microsoft/OpenAPI.NET.OData/pull/110. We'll need to test the change before closing this issue.

**Comment by maisarissi** [comment]
Hi @peombwa .

Can we validate this has been fixed and close the issue?

**Comment by peombwa** [comment]
This has been resolved as part of OData to OpenAPI v1.0.10:
``` powershell
➜ Get-Command Get-MgOrganizationBranding*

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Function        Get-MgOrganizationBranding                         1.9.6      Microsoft.Graph.Identity.DirectoryManagement
Function        Get-MgOrganizationBrandingBackgroundImage          1.9.6      Microsoft.Graph.Identity.DirectoryManagement
Function        Get-MgOrganizationBrandingBannerLogo               1.9.6      Microsoft.Graph.Identity.DirectoryManagement
Function        Get-MgOrganizationBrandingLocalization             1.9.6      Microsoft.Graph.Identity.DirectoryManagement
Function        Get-MgOrganizationBrandingLocalizationBackgroundI… 1.9.6      Microsoft.Graph.Identity.DirectoryManagement
Function        Get-MgOrganizationBrandingLocalizationBannerLogo   1.9.6      Microsoft.Graph.Identity.DirectoryManagement
Function        Get-MgOrganizationBrandingLocalizationSquareLogo   1.9.6      Microsoft.Graph.Identity.DirectoryManagement
Function        Get-MgOrganizationBrandingSquareLogo               1.9.6      Microsoft.Graph.Identity.DirectoryManagement
```
